apps/expand.go

```diff
@@ -6,29 +6,67 @@ package apps
 type ExpandLevel string
 
 const (
-	ExpandDefault = ExpandLevel("")
-	ExpandNone    = ExpandLevel("none")
-	ExpandAll     = ExpandLevel("all")
-	ExpandSummary = ExpandLevel("summary")
+	ExpandDefault ExpandLevel = ""
+	ExpandNone    ExpandLevel = "none"
+	ExpandAll     ExpandLevel = "all"
+	ExpandSummary ExpandLevel = "summary"
```

Comment:
might help detail on how `summary` differs from `all`

Reply:
It does, for each specific field.